randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -928,6 +948,11 @@ Extra - room_id: [27]
   * Layers: default
   * L0 Hatch to Reo Room/Door to Entrance Lobby Underside
   * Extra - door_idx: (63,)
+  > Pickup (Missile Tank)
+      All of the following:
+          # Currently logically useless but looks pretty frickin sweet https://youtu.be/jkn5df7GKwo
+          Level 0 Locks and Speed Booster and Shinespark Tricks (Advanced) and Disabled Door Lock Rando and Disabled Entrance Rando
+          Combat (Intermediate) or Can Kill Tough Beam-Weak Enemy
```

Comment:
what is this for?

Reply:
this is for dealing with the dessgeega and zoros in the room, I found them to be troublesome enough to have logical implications

Reply:
disagree. You can just kill the dessgeega by running into it with the shinespark, killing it instantly and stopping it from creating spikes.
The zoro you can just attack while worst case resparking a few times on the slope.